### ISSUE TITLE
vehicle saving possibly broke.

### ISSUE THREAD

**Issue by Elvensbane** [gruppe-adler/grad-persistence#23]
I updated my entire grad-presistence pack in my mission and I hadn't noticed till I followed the trail back to fn_saveVehicles.sqf being updated that it wasn't properly saving vehicles, in fact at all. Is there any way you can confirm it isn't working with the github version or maybe it was a derp on my behalf. The version from 12/15/2018 worked flawlessly when I reverted it back.

**Comment by McDiod** [comment]
No, it should work. Check [the wiki](https://github.com/gruppe-adler/grad-persistence/wiki/saving-objects) and your config, `saveVehicles = 1;` will only save editor placed vehicles now.

**Comment by Elvensbane** [comment]
I'll give it a whirl, it could be that because my descriptions.ext hadn't changed since the 2018 version.
